fluent-bundle/src/bundle.rs

```diff
@@ -167,20 +152,18 @@ impl<'bundle> FluentBundle<'bundle> {
             let message_id = &path[..ptr_pos];
             let message = self.entries.get_message(message_id)?;
             let attr_name = &path[(ptr_pos + 1)..];
-            if let Some(ref attributes) = message.attributes {
-                for attribute in attributes {
-                    if attribute.id.name == attr_name {
-                        match attribute.to_value(&env) {
-                            Ok(val) => {
-                                return Some((val.format(self), errors));
-                            }
-                            Err(err) => {
-                                errors.push(FluentError::ResolverError(err));
-                                // XXX: In the future we'll want to get the partial
-                                // value out of resolver and return it here.
-                                // We also expect to get a Vec or errors out of resolver.
-                                return Some((path.to_string(), errors));
-                            }
+            for attribute in message.attributes.iter() {
+                if attribute.id.name == attr_name {
+                    match attribute.to_value(&env) {
+                        Ok(val) => {
+                            return Some((val.format(self), errors));
+                        }
+                        Err(err) => {
+                            errors.push(FluentError::ResolverError(err));
+                            // XXX: In the future we'll want to get the partial
+                            // value out of resolver and return it here.
+                            // We also expect to get a Vec or errors out of resolver.
+                            return Some((path.to_string(), errors));
```

Comment:
Can `format_message` be removed as part of this change? It looks like `format` does everything a caller might want. If not I'll add it to my doc pr.

Reply:
It doesn't do everything. `format` allows you to get the message's value or a single attribute, but it doesn't allow you to retrieve value+attributes together.

`format_message` will be useful for higher level crates that will apply a localization message onto a widget/element.